src/Credentials.php

```diff
@@ -1,63 +1,51 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Keboola\OAuthV2Api;
 
+use InvalidArgumentException;
+
 class Credentials extends Common
 {
-    public function __construct($sapiToken, $config = [])
+    public function __construct(string $sapiToken, array $config)
     {
-        $this->client = $this->getClient(['X-StorageApi-Token' => $sapiToken], $config);
```

Comment:
ClientWrapper jsem přesunul do Common, tímpádem se to dalo trošku seškrtat na jeden konstruktor